io.embrace.sdk/Editor/EmbracePostBuildProcessor.cs

```diff
@@ -187,8 +187,10 @@ public static void OnPostprocessBuild(BuildTarget target, string pathToBuiltProj
                 // Copy run.sh script and upload binary
                 string embraceRunSHSrc = sdkDirectory + "/iOS/run.sh";
                 string embraceRunSHDest = pathToBuiltProject + "/" + EmbraceRunFileName;
-                string embraceUploadSrc = sdkDirectory + "/iOS/upload";
-                string embraceUploadDest = pathToBuiltProject + "/" + "upload";
+                
+                // TODO: We may want to deal with different file endings here. It's not vital though.
+                string embraceUploadSrc = sdkDirectory + "/iOS/embrace_symbol_upload.darwin";
```

Comment:
Is it possible to build the resultant XCode project on anything other than an Apple machine? My understanding was that that was impossible.

Reply:
Actually there's an important discussion here because I'm just unzipping the project and dropping the `.darwin` file into the Unity Package/Project. I'll look to try and unpack how this works in the AM; as far as I am currently aware the above code will handle the majority of cases though there may be issues with very specific CI pipelines.

Reply:
ya I suspect that's right and most cases this would be fine

Reply:
We'll leave it as `.darwin` then.

Reply:
@ArielDemarco ^^^

Reply:
The one we bundle in iOS seems to always be `.darwin`. I know @vitaliyf was one of the migration to store this tool as a specific asset so it's downloadable from other repos (like embrace-apple-sdk). Maybe he share some light on this specific topic.
That said, is the one we use in our [`.podpsec`](https://github.com/embrace-io/embrace-apple-sdk/blob/fe3675e8d94bb8a15106e597d3fb85078b285928/EmbraceIO.podspec#L14)

Reply:
The zip (whether you get it from downloads.embrace.io or https://github.com/embrace-io/action-symbol-upload/releases, it's the same file) has both macos and linux binaries in it. IIRC, iOS SDK would always use .darwin one, and linux ones were for RN to upload its sourcemaps.

Reply:
I'll get rid of the comment. `.darwin` will apply in all cases, as this is being added to XCode as a build phase, meaning that it will only run on macOS machines.

Is there anything I'm missing?

Reply:
Considering this is just for the build phase in Xcode, I think you're fine